### DISCUSSION TITLE
Is it possible to emulate a real FDD using a driver?

### DISCUSSION THREAD

**Discussion by NiQ1** [keirf/greaseweazle#461] [Ideas]
Do you think it's possible to write a kernel driver that communicates with Greazeweazle and presents it as a real FDD to Windows, so it can be assigned a drive letter and have IBM formatted disks natively supported?

**Comment by keirf** [comment]
Look up diskflashback by Rob Smith. This does what you ask for.

**Reply by FroOkk** [reply]
Do you know if there is anything similar for Linux?

**Reply by keirf** [reply]
There isn't exactly the same. Diskflashback could be ported onto Linux FUSE.
